tests/contrib/django/test_django_snapshots.py

```diff
@@ -156,28 +152,33 @@ def test_psycopg_query_default(client, psycopg2_patched):
 
 
 @pytest.mark.skipif(django.VERSION < (3, 0, 0), reason="ASGI not supported in django<3")
-@pytest.mark.snapshot(
+@snapshot(
     variants={
         "30": (3, 0, 0) <= django.VERSION < (3, 1, 0),
         "31": (3, 1, 0) <= django.VERSION < (3, 2, 0),
         "3x": django.VERSION >= (3, 2, 0),
     },
+    token_override="tests.contrib.django.test_django_snapshots.test_asgi_200",
```

Comment:
this works along with the variants as well? that is we get `30`/`31`/`3x` names files as well?

Reply:
Yes the variants are added at the end of the token name